app/views/layouts/application.html.erb

```diff
@@ -1,15 +1,16 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <title>SampleApp</title>
-    <%= csrf_meta_tags %>
-    <%= csp_meta_tag %>
 
-    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
-    <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
-  </head>
+<head>
+  <title><%= yield(:title) %> | Rails Tutorials</title>
```

Comment:
` Rails Tutorials`
tìm hiểu và sử dụng I18n nhé

=> DÙNG I18n cho TOÀN BỘ TEXT TRONG HTML FILE NHÉ